router/core/graph_server.go

```diff
@@ -600,8 +591,8 @@ func (s *graphServer) buildGraphMux(ctx context.Context,
 	featureFlagName string,
 	routerConfigVersion string,
 	engineConfig *nodev1.EngineConfiguration,
-	configSubgraphs []*nodev1.Subgraph) (*graphMux, error) {
-
+	configSubgraphs []*nodev1.Subgraph,
```

Comment:
nit: This looks a bit odd
```suggestion
	configSubgraphs []*nodev1.Subgraph
```

Reply:
This is the go formatter, I didn't even modify this function, just saved the file. `gofmt` and `gofumpt` both force this form.